.github/workflows/tutorial.yml

```diff
@@ -0,0 +1,91 @@
+name: Tutorials
+
+env:
+  IMAGE_NAME: activemq-artemis-operator
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+on:
+  pull_request:
+  schedule:
+    - cron: '0 0 * * 0'
+  workflow_dispatch:
+    inputs:
+      skipTutorials:
+        description: 'Skip Tutorials'
+        required: false
+        default: false
+        type: boolean
+
+jobs:
+  run-tutorials:
+    name: Run Tutorials
+    runs-on: ubuntu-latest
+    services:
+      registry:
+        image: registry:2
+        ports:
+          - 5000:5000
+    steps:
+      - name: Set up env
+        run: |
+          echo "HOSTNAME=${HOSTNAME}" >> ${GITHUB_ENV}
+
+      - name: Set up Docker
+        uses: docker/setup-docker-action@v4
+        with:
+          daemon-config: |
+            {
+              "insecure-registries" : [ "${{env.HOSTNAME}}:5000" ]
+            }
+      - name: Checkout the repo
+        uses: actions/checkout@v4
+
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: '1.23.9'
+          cache: true
+
+      - name: Install markdown-runner
+        run: go install github.com/arkmq-org/markdown-runner@latest
+
+      - name: Check for changed tutorial files
+        id: changed-tutorial-files
+        if: github.event_name == 'pull_request'
+        uses: tj-actions/changed-files@v44
+        with:
+          files: docs/tutorials/**
+
+      - name: Get latest release tag with gh CLI
+        if: ${{ (github.event_name == 'schedule' || (github.event_name == 'workflow_dispatch' && !inputs.skipTutorials)) }}
+        id: get_tag
+        run: |
+          LATEST_TAG=$(gh release list --limit 1 --json tagName --jq .[0].tagName)
+          echo "tag=${LATEST_TAG}" >> $GITHUB_OUTPUT
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Run the tutorials for the latest release build
+        if: ${{ (github.event_name == 'schedule' || (github.event_name == 'workflow_dispatch' && !inputs.skipTutorials)) }}
+        run: |
+          echo "Checking out latest release tag: ${{ steps.get_tag.outputs.tag }}"
```

Comment:
Why checking out the latest release tag? Should the current branch/tag (the one selected before clicking the button `Run workflow` or the main for scheduled executions) be checked?